src/unittests/f32_interval.spec.ts

```diff
@@ -0,0 +1,439 @@
+    { bounds: [kValue.f32.negative.max, 0], value: kValue.f32.subnormal.positive.min, expected: false },
+    { bounds: [kValue.f32.negative.max, 0], value: kValue.f32.subnormal.positive.max, expected: false },
+    { bounds: [kValue.f32.negative.max, 0], value: kValue.f32.subnormal.negative.min, expected: true },
+    { bounds: [kValue.f32.negative.max, 0], value: kValue.f32.subnormal.negative.max, expected: true },
+    { bounds: [0, kValue.f32.subnormal.positive.min], value: kValue.f32.subnormal.positive.min, expected: true },
+    { bounds: [0, kValue.f32.subnormal.positive.min], value: kValue.f32.subnormal.positive.max, expected: false },
+    { bounds: [0, kValue.f32.subnormal.positive.min], value: kValue.f32.subnormal.negative.min, expected: false },
+    { bounds: [0, kValue.f32.subnormal.positive.min], value: kValue.f32.subnormal.negative.max, expected: false },
+    { bounds: [kValue.f32.subnormal.negative.max, 0], value: kValue.f32.subnormal.positive.min, expected: false },
+    { bounds: [kValue.f32.subnormal.negative.max, 0], value: kValue.f32.subnormal.positive.max, expected: false },
+    { bounds: [kValue.f32.subnormal.negative.max, 0], value: kValue.f32.subnormal.negative.min, expected: false },
+    { bounds: [kValue.f32.subnormal.negative.max, 0], value: kValue.f32.subnormal.negative.max, expected: true },
+    ]
+  )
+  .fn(t => {
+    const i = arrayToInterval(t.params.bounds);
+    const value = t.params.value;
+    const expected = t.params.expected;
+
+    const got = i.contains(value);
+    t.expect(expected === got, `${i}.contains(${value}) returned ${got}. Expected ${expected}`);
+  });
+
+interface ContainsIntervalCase {
+  lhs: [number, number];
+  rhs: [number, number];
+  expected: boolean;
+}
+
+g.test('contains_interval')
+  .paramsSubcasesOnly<ContainsIntervalCase>(
+    // prettier-ignore
+    [
+      // Common usage
+      { lhs: [-10, 10], rhs: [0, 0], expected: true},
+      { lhs: [-10, 10], rhs: [-1, 0], expected: true},
+      { lhs: [-10, 10], rhs: [0, 2], expected: true},
+      { lhs: [-10, 10], rhs: [-1, 2], expected: true},
+      { lhs: [-10, 10], rhs: [0, 10], expected: true},
+      { lhs: [-10, 10], rhs: [-10, 2], expected: true},
+      { lhs: [-10, 10], rhs: [-10, 10], expected: true},
+      { lhs: [-10, 10], rhs: [-100, 10], expected: false},
+
+      // Upper infinity
+      { lhs: [0, kValue.f32.infinity.positive], rhs: [0, 0], expected: true},
+      { lhs: [0, kValue.f32.infinity.positive], rhs: [-1, 0], expected: false},
+      { lhs: [0, kValue.f32.infinity.positive], rhs: [0, 1], expected: true},
+      { lhs: [0, kValue.f32.infinity.positive], rhs: [0, kValue.f32.infinity.positive], expected: true},
+      { lhs: [0, kValue.f32.infinity.positive], rhs: [100, kValue.f32.infinity.positive], expected: true},
+      { lhs: [0, kValue.f32.infinity.positive], rhs: [kValue.f32.infinity.negative, kValue.f32.infinity.positive], expected: false},
```

Comment:
What about:
`lhs: [0, kValue.f32.positive.max], rhs: [0, kValue.f32.infinity.positive]`
and:
`lhs: [0, kValue.f32.infinity.positive], rhs: [0, kValue.f32.positive.max]`

Reply:
Looks good, thanks